docs/quickstart.md

```diff
@@ -0,0 +1,177 @@
+curl -fsSL -o hello-vmlinux.bin https://s3.amazonaws.com/spec.ccfc.min/img/hello/kernel/hello-vmlinux.bin
+curl -fsSL -o hello-rootfs.ext4 https://s3.amazonaws.com/spec.ccfc.min/img/hello/fsfiles/hello-rootfs.ext4
+
+# Inject the agent, runc, and a startup script into the VM image
+mkdir /tmp/mnt
+# Construct fc-agent.start
+cat >fc-agent.start <<EOF
+#!/bin/sh
+mkdir -p /container
+exec > /container/agent-debug.log # Debug logs from the agent
+exec 2>&1
+touch /container/runtime # TODO determine what this is for
+mkdir /container/rootfs
+mount -t auto -o rw /dev/vdb /container/rootfs
+cd /container
+/usr/local/bin/agent -id 1 -debug &
+EOF
+chmod +x fc-agent.start
+truncate --size=+50M hello-rootfs.ext4
+e2fsck -f hello-rootfs.ext4
+resize2fs hello-rootfs.ext4
+sudo mount hello-rootfs.ext4 /tmp/mnt
+sudo cp $(which runc) firecracker-containerd/agent/agent /tmp/mnt/usr/local/bin
+sudo cp fc-agent.start /tmp/mnt/etc/local.d
+sudo ln -s /etc/init.d/local /tmp/mnt/etc/runlevels/default/local
+sudo ln -s /etc/init.d/cgroups /tmp/mnt/etc/runlevels/default/cgroups
+sudo umount /tmp/mnt
+rmdir /tmp/mnt
+
+cd ~
+
+# Configure containerd to use our new snapshotter
+sudo mkdir -p /etc/containerd
+sudo tee -a /etc/containerd/config.toml <<EOF
+[proxy_plugins]
+  [proxy_plugins.firecracker-naive]
+    type = "snapshot"
+    address = "/var/run/firecracker-containerd/naive-snapshotter.sock"
+EOF
+
+cd ~
+
+# Configure the aws.firecracker runtime
+sudo mkdir -p /var/lib/firecracker-containerd/runtime
+sudo cp hello-rootfs.ext4 hello-vmlinux.bin /var/lib/firecracker-containerd/runtime
+sudo mkdir -p /etc/containerd
+sudo tee -a /etc/containerd/firecracker-runtime.json <<EOF
+{
+  "firecracker_binary_path": "/usr/local/bin/firecracker",
+  "socket_path": "./firecracker.sock",
+  "kernel_image_path": "/var/lib/firecracker-containerd/runtime/hello-vmlinux.bin",
+  "kernel_args": "console=ttyS0 noapic reboot=k panic=1 pci=off nomodules rw",
+  "root_drive": "/var/lib/firecracker-containerd/runtime/hello-rootfs.ext4",
+  "cpu_count": 1,
+  "cpu_template": "T2",
+  "console": "stdio",
+  "log_fifo": "/tmp/fc-logs.fifo",
+  "log_level": "Debug",
+  "metrics_fifo": "/tmp/fc-metrics.fifo"
+}
+EOF
+```
+
+4. Open a new terminal and start the `naive_snapshotter` program in the
+   foreground
+
+```bash
+sudo mkdir -p /var/run/firecracker-containerd /var/lib/firecracker-containerd/naive
+sudo /usr/local/bin/naive_snapshotter -address /var/run/firecracker-containerd/naive-snapshotter.sock -path /var/lib/firecracker-containerd/naive -debug
+```
+
+5. Open a new terminal and start `containerd` in the foreground
+
+```bash
+sudo PATH=$PATH /usr/local/bin/containerd
+```
+
+6. Open a new terminal, pull an image, and run a container!
+
+```bash
+sudo /usr/local/bin/ctr image pull --snapshotter firecracker-naive docker.io/library/debian:latest
+sudo /usr/local/bin/ctr run --snapshotter firecracker-naive --runtime aws.firecracker --tty docker.io/library/debian:latest test
+```
```

Comment:
Is it worth adding a note at the end to remember to terminate/halt the instance when done?